src/components/AppNavigationToggle/AppNavigationToggle.vue

```diff
@@ -25,12 +25,15 @@
 	<a class="app-navigation-toggle"
 		href="#"
 		:aria-expanded="open ? 'true' : 'false'"
+		:aria-label="open ? t('Close navigation') : t('Open navigation')"
```

Comment:
I just remembered that there seems to be an issue extracting the translation when it's within a html-property: https://github.com/nextcloud/nextcloud-vue/pull/1776

So it might be better to make it a computed property instead.

Reply:
```suggestion
		:aria-label="ariaLabel"
```

Reply:
Thanks @raimund-schluessler, added both changes! :)